collated/main/A0153440R.md

```diff
@@ -0,0 +1,806 @@
+# A0153440R
+###### \java\jym\manager\logic\commands\AddCommand.java
+``` java
+    public AddCommand(String description, Object ... objects) throws IllegalValueException{
+    	if(objects.length > 3){//f**k in this case - this should never happen b/c we control parser. max args is date(s), location, priority
+    		throw new IllegalArgumentException();
+    	}
+    	boolean isTask = true; //by default
+    	for(Object o : objects){
+    		if(o instanceof List){
```

Comment:
Indentation problem here. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.